devices/common_patches/quadspi/dr_multi_access.yaml

```
@@ -0,0 +1,17 @@
+"QUADSPI":
+  DR:
+    _delete:
+      - DATA
+    _add:
+      DataOneByte:
+        description: One byte access to DATA field
+        bitOffset: 0
+        bitWidth: 8
+      DataHalfWord:
+        description: Half word (16 bytes) access to DATA field
+        bitOffset: 0
+        bitWidth: 16
+      DataFullWord:
+        description: Full word (32 bytes) access to DATA field
+        bitOffset: 0
+        bitWidth: 32
```

I suppose you are trying to do volatile access to lower bytes of register accordind to:
> Word, halfword, and byte accesses to this register are supported.

In this case you need to add additional registers, not the fields. Also don't rename registers/fields if it is not really needed.
```suggestion
  _modify:
    DR:
      description: "Data register: full word (32 bytes) access" 
  _add:
    DR16:
      addressOffset: 0x20
      alternateRegister: DR
      size: 0x10
      description: "Data register: half word (16 bit) access" 
      fields:
        DATA:
          description: Data
          bitOffset: 0
          bitWidth: 16
    DR8:
      addressOffset: 0x20
      alternateRegister: DR
      size: 0x08
      description: "Data register: one byte (8 bit) access" 
      fields:
        DATA:
          description: Data
          bitOffset: 0
          bitWidth: 8
```

and in `base.yaml`:
```yaml
  DR:
    DATA: [0, 0xFFFF_FFFF]

  DR16:
    DATA: [0, 0xFFFF]

  DR8:
    DATA: [0, 0xFF]
```

---

thanks for helping!